internal/connect.go

```diff
@@ -32,7 +34,10 @@ import (
 
 var InvalidStateErr = errors.New("invalid new state")
 
-var client *hazelcast.Client
+var (
+	client    *hazelcast.Client
+	sqlDriver *sql.DB
```

Comment:
Should these global variables protected against concurrent access?

Reply:
No they are not, but because of the nature of CLI, only 1 command tries to access them in any point in time. I can introduce them if you'd like, but didn't think it was necessary so far

Reply:
OK. Could you add a TODO to protect those? In the later releases we may introduce things like background jobs.